app/javascript/github_grass.js

```diff
@@ -22,6 +22,9 @@ document.addEventListener('DOMContentLoaded', () => {
         .catch((error) => {
           console.warn(error)
         })
+        .then(() => {
+          document.querySelector('.js-user-grass__loading').remove()
```

Comment:
そこまで気にしなくても良さそうですが、「`.js-user-grass__loading`というクラスを持った最初の要素を選択する」というのがちょっと気になりました。

`user-grass****`というクラスは学習時間の草でも使われていてGitHubの草専用ではなさそうに見えるので、仮に将来`.js-user-grass__loading`を使う他のローディング画像がダッシュボードに追加されたら、意図しない動作が発生して調査が必要になるといったことが起こりそうだと思いました。

↓のようにするとGitHubの草カードの中のローディング画像を特定できるようになります。
```suggestion
          document.querySelector('.js-github-grass + .js-user-grass__loading').remove()
```

とはいえ現状では`.js-user-grass__loading`を持った要素は1つしかないので、今のコードの方がシンブルで良いようにも思えます。
ですので、tomonariさんの意図と「`.js-user-grass__loading`というクラスを持った最初の要素を選択する」という動作が一致しているのであれば問題ないのかなと思いました。
提案まで行かない情報提供程度に受け止めていただければ幸いです🙏

[FYI] querySelectorで使えるCSSセレクタは、以下の項目が参考になりそうです。
https://developer.mozilla.org/ja/docs/Learn/CSS/Building_blocks/Selectors#reference_table_of_selectors

Reply:
なるほど、`user-grass`はパーツの使い回しのためか学習時間の草と共通になっているんですね！
これは見落としていました🙇‍♂️
Githubの草専用だと思って実装してたので、ご提案の通りに修正しました。[a8e12b1](https://github.com/fjordllc/bootcamp/pull/5218/commits/a8e12b10fd399e6bc1b5e82f20edf75c9a4b767e)